source/logHandler.py

```diff
@@ -249,7 +249,7 @@ class RemoteHandler(logging.Handler):
 
 	def __init__(self):
 		#Load nvdaHelperRemote.dll but with an altered search path so it can pick up other dlls in lib
-		path=os.path.abspath(os.path.join(u"lib",buildVersion.version,u"nvdaHelperRemote.dll"))
+		path = os.path.join(globalVars.appDir, "lib", buildVersion.version, "nvdaHelperRemote.dll")

```

Comment:
`RemoteHandler` is used in nvda_slave and slave does not have access to `globalVars` currently. Perhaps `globalVars.appDir` should be set to `None` in `globalVarrs by default and set to NVDA directory when they're imported for the first time.